src/dbms/database.hpp

```diff
@@ -156,7 +160,8 @@ class Database {
   query::stream::Streams streams_;                  //!< Streams associated with the storage
 
   // TODO: Move to a better place
-  utils::SkipList<query::PlanCacheEntry> plan_cache_;  //!< Plan cache associated with the storage
+  utils::Synchronized<utils::LRUCache<uint64_t, std::shared_ptr<query::CachedPlan>>, utils::SpinLock>
+      plan_cache_;  //!< Plan cache associated with the storage
```

Comment:
Consider creating a type for this method, i.e.
`using PlanCacheLRU = utils::Synchronized<utils::LRUCache<uint64_t, std::shared_ptr<query::CachedPlan>>, utils::SpinLock>;`